.yamato/config/vfx_urp.metafile

```diff
@@ -17,19 +17,19 @@ platforms:
       - name: mono
     color_spaces:
       - Linear
-  - name: OSX 
-    apis:
-      - name: Metal
-        exclude_test_platforms:
-          - name: playmode_XR
-      - name: OpenGLCore
-        exclude_test_platforms:
-          - name: playmode_XR
-          - name: Standalone
-    build_configs:
-      - name: mono
-    color_spaces:
-      - Linear
+#  - name: OSX 
```

Comment:
Is it necessary? from what I understand that would also remove the jobs for manual execution.
There seems to be no osx jobs in the vfx ABV

Reply:
ah my mistake, thanks!